Header header
float64[] opt_trajectory
float64[3] final_pos
